docs/content/en/docs/implementing/otel.md

```diff
@@ -4,26 +4,193 @@ description: How to standardize access to OpenTelemetry observability data
 weight: 140
 ---
 
+
+The Keptn Lifecycle Toolkit (KLT) makes any Kubernetes deployment observable.
+In other words, it creates a distributed, end-to-end trace
+of everything Kubernetes does in the context of a Deployment.
+It provides this information
+for all applications running in your cluster,
+and includes information about
+everything Kubernetes does in the context of a deployment.
```

Comment:
I would remove these 4 lines since it says again what was said few lines above

Reply:
done